packages/table/src/headers/columnHeader.tsx

```diff
@@ -60,6 +65,12 @@ export interface IColumnHeaderProps extends ISelectableProps, IColumnIndices, IC
     viewportRect: Rect;
 
     /**
+     * Enables/disables the reordering interaction.
+     * @default true
+     */
+    isReorderable?: boolean;
```

Comment:
Oh gosh, I don't know of a case where a single column would need to disable reordering (unless it were a frozen column, which I don't know if we support yet?). Would prefer to keep it all-or-nothing for now.